### DISCUSSION TITLE
null or zero?

### DISCUSSION THREAD

**Discussion by silicahd** [alpinejs/alpine#1137] [4. General]
I have this expression 
```
     :class="fr ? 'bg-blue-400' : 'bg-red-400'"
````

I pass it a 0 or 1. However it only seems to work with true or null. how can I get this to work with 0 or 1. I know this is stupid but I think my brain is fried at the moment.

**Comment by silicahd** [comment]
Got it 

```
:class="{ 'bg-blue-400': fr == 1, 'bg-red-400': fr == 0 }"
```

**Comment by KevinBatdorf** [comment]
`fr` is likely a string, so the string `'0'` is a truthy value. 

You could also use `:class="+fr ? 'bg-blue-400' : 'bg-red-400'"`, I think, but your solution is good too. 

Or `:class="fr == 1 ? 'bg-blue-400' : 'bg-red-400'"`
(the important part being `==` instead of `===` which will convert the string numbers into numbers, then compare)
